src/twitch.rs

```diff
@@ -27,44 +24,48 @@ pub async fn twitch_irc(config: &CompleteConfig, tx: Sender<Data>, mut rx: Recei
     let mut stream = client.stream().unwrap();
 
     loop {
-        if let Some(Some(Ok(message))) = unconstrained(stream.next()).now_or_never() {
-            match message.command {
-                Command::PRIVMSG(ref _target, ref msg) => {
-                    let user = match message.source_nickname() {
-                        Some(username) => username.to_string(),
-                        None => "Undefined username".to_string(),
-                    };
-                    tx.send(Data::new(
-                        Local::now()
-                            .format(config.frontend.date_format.as_str())
-                            .to_string(),
-                        user,
-                        msg.to_string(),
-                        false,
-                    ))
-                    .await
-                    .unwrap();
-                }
-                Command::NOTICE(ref _target, ref msg) => {
-                    tx.send(Data::new(
-                        Local::now()
-                            .format(config.frontend.date_format.as_str())
-                            .to_string(),
-                        "twitch".to_string(),
-                        format!("NOTICE: {}", msg),
-                        true,
-                    ))
-                    .await
-                    .unwrap();
-                }
-                _ => (),
-            }
-        }
+        tokio::select! {
+            biased;
```

Comment:
What does `biased;` do?

Reply:
It lets me [https://docs.rs/tokio/1.12.0/tokio/macro.select.html#fairness](control polling order). This is partly for ease of mind so we always poll for a new message to send to Twitch before polling for new messages from Twitch.

Reply:
Very well.